### PR TITLE
add ability to focus component

### PR DESCRIPTION
**Pull request by austinpgraham**
Hello all! Great library. I've made a small change that allows for the execution of an onFocus of a text input, given some requirements from the Android app at Tailwind. Please excuse the small amounts of prettifying in the file as well.

### CODE REVIEW

File: src/TestScope.js [hunk 1]
```diff
@@ -128,6 +132,18 @@ export default class TestScope {
...
+  // Your component should respond to the property `onFocus`.
+  //
+  // identifier - Identifier for the component.
+  //
+  // Returns a promise, use await when calling this function. Promise will be
+  // rejected if the component is not found.
+  async focus(identifier) {
+    const component = await this.findComponent(identifier);
+    component.props.onFocus();
+  }
```
Comment (austinpgraham) [inline src/TestScope.js]: This function does exactly the same thing as press, but rather focuses the component 😄

**Comment by AbigailMcP** [comment]
Thanks @austinpgraham, looks good!

To get this merged, we need to tackle three things:
1. Add a test to the sample app in this repo.
1. Add typings for the new `focus` method to [Cavy's Typescript type definitions in DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/cb5ee9a06072df4d681dcb14a2e674e18f45319a/types/cavy)
2. Add it to our [documentation](https://cavy.app/) by opening a PR on [this repo](https://github.com/pixielabs/cavy-app).

Would you be happy tackling any/all of these? Let me know :)

Thanks again for contributing to Cavy! ❤️

**Comment by austinpgraham** [comment]
@AbigailMcP I'd love to! I'll try to knock them out in the next couple of days

**Comment by austinpgraham** [comment]
@AbigailMcP I have the relevant PRs here:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41474
https://github.com/pixielabs/cavy-app/pull/6

I have also updated this one with a test in the sample app. However, I didn't immediately see a way to to test it meaningfully without adding new functionality to the app. Would appreciate some guidance!

Thanks ❤️

**Comment by AbigailMcP** [comment]
Thanks @austinpgraham ! We're currently thinking about our testing (see PR https://github.com/pixielabs/cavy/pull/147) so your test is great for now 👌 

I'll hopefully get round to merging at the end of today! Thanks for the docs and types too :)

**Pull request merged**